src/RFC-0120_Consensus.md

```diff
@@ -360,7 +360,7 @@ The block MUST be transmitted in canonical ordering. The advantage of this appro
 done by the whole network, and verification of sorting is exceptionally cheap.
 
 Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. `\\(k \cdot G + v \cdot H\\)`.
-Transaction inputs are sorted lexicographically by the [hash of the output](RFC-0121_ConsensusEncoding.md#transaction-output) that is spent by the input.
+Transaction inputs are sorted lexicographically by the [hash of the output](RFCD-0121_ConsensusEncoding.md#transaction-output) that is spent by the input.
```

Comment:
This should be updated to what the code is doing. Might be unchanged, but then remove the link to a deprecated RFC.

Reply:
I've removed 2 links pointing to this deprecated document, here and from the RFC-0131/Mining.
Will have to update https://rfc.tari.com/RFC-0131_Mining.html once new implementation starts working.